### ISSUE TITLE
Use +[NSLocale preferredLanguages] instead of +[NSLocale currentLocale] in default user agent string

### ISSUE THREAD

**Issue by rbsgn** [pokeb/asi-http-request#50]
`+currentLocale` returns user's locale overlaid with other language settings on the iPhone. For some reason I'm getting en_US on my iPhone even when I've set up Russian in Settings application. `[[NSLocale preferredLanguages] objectAtIndex:0]` seem solves this problem.

**Comment by pokeb** [comment]
According to http://stackoverflow.com/questions/1522210/nslocale-currentlocale-always-returns-en-us-not-users-current-language, the locale you get is dependent on the region you have set, not the language. Is your region set to USA?

[[NSLocale preferredLanguages] objectAtIndex:0] gives me 'en' rather than 'en_GB', while Safari gives me 'en-us' - I suspect this is because Safari doesn't have a UK english localisation.

I don't know if there's a reliable way to get both country and language - any suggestions?

**Comment by rbsgn** [comment]
Yes, I've set up US region format. Unfortunately, I don't know how to get both country and language :-(
